data-structures-stacks/get-top.js

```diff
@@ -0,0 +1,7 @@
+/* exported getTop */
+
+function getTop(stack) {
+  if (stack.length !== 0) {
```

Comment:
There is no `length` property on stacks. You don't need this check here, you can just return `stack.peek()`